WolvenKit.Modkit/RED4/Tasks/WwiseTask.cs

```diff
@@ -0,0 +1,47 @@
+using System.IO;
+using WolvenKit.Core.Wwise;
+
+namespace CP77Tools.Tasks
+{
+    public partial class ConsoleFunctions
+    {
+        #region Methods
+
+        public int WwiseTask(string path, string outpath, bool wem)
+        {
+            if (string.IsNullOrEmpty(path))
+            {
+                return 0;
+            }
+
+            if (string.IsNullOrEmpty(outpath))
+            {
+                outpath = Path.ChangeExtension(path, ".ogg");
+            }
+
+            if (wem)
+            {
+                var file = File.ReadAllBytes(path);
+                using var ms = new MemoryStream(file);
+                using var br = new BinaryReader(ms);
+
+                var inBuffer = br.ReadBytes(file.Length);
```

Comment:
As far as I can see you just need `var inbuffer = File.ReadAllBytes(path);` here. No need for streams if you just use the byte array in the end right?

Reply:
Ah yeah sorry, I had that in my test but for this file I'd copied the Kraken stuff. I'll fix it ASAP.